coldfront/core/user/templates/user/user_update_departments.html

```diff
@@ -0,0 +1,49 @@
+{% extends "common/base.html" %}
+{% load feature_flags %}
+{% load crispy_forms_tags %}
+{% load static %}
+
+
+{% block title %}
+Update {{ department_display_name }}s
+{% endblock %}
+
+{% block head %}
+{{ wizard.form.media }}
+{% endblock %}
+
+{% block content %}
+
+<h1>Update Departments</h1>
+<hr>
+
+<script type="text/javascript" src="{% static 'common/js/leave_form_alert.js' %}"></script>
+<script type='text/javascript' src="{% static 'selectize/selectize.min.js' %}"></script>
+<link rel='stylesheet' type='text/css' href="{% static 'selectize/selectize.bootstrap3.css' %}"/>
+
+<p>Select all the {{ department_display_name | lower }}s you are associated with.
+Unselect a {{ department_display_name | lower }} to remove it from your associations.</p>
```

Comment:
We're using "Deselect" in one other place in the UI, so let's use it here too instead of "Unselect".